debs/SPECS/3.10.0/wazuh-agent/debian/postrm

```diff
@@ -55,6 +49,12 @@ case "$1" in
     ;;
 
     purge)
+        if getent passwd | grep -q "^ossec" ; then
```

Comment:
This change is ok but the indentation in some lines are tabs and in other are spaces. Can you change them to spaces?

Reply:
Done  in https://github.com/wazuh/wazuh-packages/pull/227/commits/c39e24454ceb2e58427226d03222a53b6d0e91eb